echo-scheduler/src/main/groovy/com/netflix/spinnaker/echo/config/SchedulerConfiguration.groovy

```diff
@@ -124,13 +90,33 @@ class SchedulerConfiguration {
   @Bean
   JobDetailFactoryBean pipelineJobBean() {
     JobDetailFactoryBean triggerJob = new JobDetailFactoryBean()
-    triggerJob.jobClass = PipelineTriggerJob.class
-    triggerJob.name = TriggerConverter.JOB_ID
-    triggerJob.durability = true
+    triggerJob.setJobClass(PipelineTriggerJob.class)
+    triggerJob.setName(TriggerConverter.JOB_ID)
+    triggerJob.setDurability(true)
 
     return triggerJob
   }
 
+  @Bean
+  SchedulerFactoryBeanCustomizer echoSchedulerFactoryBeanCustomizer(
+    Optional<DataSource> dataSourceOptional,
+    TriggerListener triggerListener
+  ) {
+    return new SchedulerFactoryBeanCustomizer() {
+      @Override
+      void customize(SchedulerFactoryBean schedulerFactoryBean) {
+        if (dataSourceOptional.isPresent()) {
+          schedulerFactoryBean.setDataSource(dataSourceOptional.get())
+        }
+
+        Properties props = new Properties()
+        props.put("org.quartz.jobStore.isClustered", "true")
```

Comment:
couldn't figure out a better way to do this... if I make a new instance of `QuartzProperties` the app fails to start due to multiple bean

Reply:
I don’t follow? Why would you need a new `QuartzProperties`?

Reply:
to specify customization for the quartz scheduler... 
If you think this is ok then certainly happy to keep as is

Reply:
I don’t think that will be necessary. Since we’re using Spring Boot for autoconfiguration, we’ll actually be exposing `spring.quartz.*` configuration options to operators automagically!